api/src/main/java/org/pmiops/workbench/monitoring/StackdriverStatsExporterService.java

```diff
@@ -38,6 +38,14 @@ public void createAndRegister() {
             StackdriverStatsConfiguration.builder()
                 .setMetricNamePrefix(buildMetricNamePrefix())
                 .setProjectId(workbenchConfigProvider.get().server.projectId)
+                .setMonitoredResource(
+                    MonitoredResource.newBuilder()
```

Comment:
These are placeholder values. I thought i could implement all of https://precisionmedicineinitiative.atlassian.net/browse/RW-4226 in this PR, but looks like there's going to be a new dependency on an AppEngine service.